Makefile

```diff
@@ -137,7 +137,15 @@ go-vet:
 
 .PHONY: go-lint
 go-lint: cmd/cli/chart.tgz pkg/envoy/lds/stats.wasm
-	go run github.com/golangci/golangci-lint/cmd/golangci-lint run --config .golangci.yml
+	@if ! golangci-lint version 2>&1 | grep 1.30; then \
+		echo Installing golangci-lint; \
+		go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.30; \
+	fi; \
+	golangci-lint run \
```

Comment:
What if there are multiple versions installed already?

Reply:
It looks like `go install` will just overwrite the bin with the desired version. Perhaps the better approach is to use a [golang-ci container](https://golangci-lint.run/usage/install/#docker) to accomplish this. I personally don't mind the overwriting behavior, but I can understand if that's not desirable for other devs or just by principle alone.

Reply:
I worry overwriting the existing version will make IDE managed environments unhappy when such linters are installed and managed by IDEs. Is there a way to avoid overwriting and reference the version specific binary directly?

Reply:
It looks like this isn't possible with `go install`. I think the next best thing is to go the container approach to prevent this overwriting issue.

Reply:
I didn't mean for this to be a blocker. Maybe add a note in the script and provide a flag so the overwriting can be optional?

Reply:
I think the container implementation is better. We already expect docker on our local machines, easiest to use the version based off of an image. I also took the opportunity to start using golangci-lint 1.41.1.